### PR TITLE
Enhancement/skale 2097 Update to skale.py v3

### PR DESCRIPTION
**Pull request by alexgex**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -23,6 +23,7 @@ jobs:
         - "flake8 ."
       script:
       - export PYTHONPATH=$PYTHONPATH:.
+      - python tests/integration/prepare_validator.py # patch - remove after fix in SKALE manager
```
Comment (dmytrotkk) [inline .travis.yml]: Remove comment

File: bounty/bounty_agent.py [hunk 2]
```diff
@@ -126,26 +124,26 @@ def get_bounty(self):
 
         self.logger.info(LONG_DOUBLE_LINE)
 
-        if receipt['status'] == 1:
+        if res_tx.receipt['status'] == 1:
```
Comment (dmytrotkk) [inline bounty/bounty_agent.py]: this check is reduntant with current `transaction_method` implementation

**Review by dmytrotkk — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (alexgex) [reply .travis.yml]: removed

File: bounty/bounty_agent.py (hunk 2 above)
Comment (alexgex) [reply bounty/bounty_agent.py]: It was left for future changes in transaction_method in order to not forget about it) 
Anyway - removed

**Review by dmytrotkk — APPROVED** [review]

**Pull request merged**